plugins/org.jkiss.dbeaver.ext.oracle/src/org/jkiss/dbeaver/ext/oracle/model/OracleTablespace.java

```diff
@@ -297,6 +297,7 @@ public DBSObject refreshObject(@NotNull DBRProgressMonitor monitor) throws DBExc
         usedSize = null;
         fileCache.clearCache();
         segmentCache.clearCache();
+        loadSizes(monitor);
```

Comment:
Just nullify cached sizes. Otherwise refresh of multiple tablespaces may take a lof of time